### DISCUSSION TITLE
Tutorial - Where I put command ?

### DISCUSSION THREAD

**Discussion by Mara-Li** [SilentVoid13/Templater#182] [Help] [answered]
Hello.
I think it's a stupid question (because, well, I didn't found the response ??). 
I want to use some templater command created by other, but I really doesn't know WHERE I need to put the command. 

Anyone can explain to me how I use this plugin? 

I understand that some templates are replaced by this, but internal  custom command? How it's work? How can I use it?

**Comment by lucasew** [comment, marked as answer]
You use the commands on the template note. It runs and expands once when you create a note based on a template.
